mcs/class/System.Drawing/Test/System.Drawing.Drawing2D/PathGradientBrushTest.cs

```diff
@@ -36,7 +36,9 @@
 namespace MonoTests.System.Drawing.Drawing2D {
 
 	[TestFixture]
+#if !NET40
 	[SecurityPermission (SecurityAction.Deny, UnmanagedCode = true)]
+#endif
```

Comment:
We're using the `NET_4_0` symbol in the codebase, but given that CAS was never fully implemented on Mono the easier/cleaner solution is just to remove the attributes.